src/System.Security.Cryptography.X509Certificates/tests/FindTests.cs

```diff
@@ -369,13 +369,33 @@ public static void TestBySubjectKeyIdentifier_MatchA()
                 "B4D738B2D4978AFF290A0B02987BABD114FEE9C7");
         }
 
-        [Fact]
+        [Theory]
+        [InlineData("5971A65A334DDA980780FF841EBE87F9723241F2")]
+        // Whitespace is allowed
+        [InlineData("59 71\tA6 5A 33 4D DA 98 07 80 FF 84 1E BE 87 F9 72 32 41 F2")]
```

Comment:
Maybe add an example of every whitespace character (U+0009, U+000A, U+000B, U+000C, U+000D, U+0020, U+0085, U+00A0, U+1680, U+2000, U+2001, U+2002, U+2003, U+2004, U+2005, U+2006, U+2007, U+2008, U+2009, U+200A, U+2028, U+2029, U+202F, U+205F, U+3000) to catch regressions that breaks support for one of the more obscure cases.